webpack/components/extensions/HostDetails/Tabs/DebsTab/DebInstallModal.scss

```
@@ -0,0 +1,3 @@
+#package-install-modal .pf-c-input-group input {
```

```suggestion
#deb-install-modal .pf-c-input-group input {
```

If we don't want this CSS to also affect PackageInstallModal, maybe it's better to use a distinct ID.